From: Emmanuele Bassi <ebassi@gnome.org>
Date: Thu, 30 Sep 2021 00:38:17 +0100
Subject: docs: Fix gtk_tree_selection_get_selected_rows()

Reword the description and return value so that they look better on
docs.gtk.org.

Origin: upstream, 3.24.31, commit:1900fc2f80e9bcf020ed48855e7bb763a1d8aebe
---
 gtk/gtktreeselection.c | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/gtk/gtktreeselection.c b/gtk/gtktreeselection.c
index eb69a35..677a277 100644
--- a/gtk/gtktreeselection.c
+++ b/gtk/gtktreeselection.c
@@ -541,9 +541,12 @@ gtk_tree_selection_get_selected (GtkTreeSelection  *selection,
  * @selection: A #GtkTreeSelection.
  * @model: (out) (allow-none) (transfer none): A pointer to set to the #GtkTreeModel, or %NULL.
  *
- * Creates a list of path of all selected rows. Additionally, if you are
- * planning on modifying the model after calling this function, you may
- * want to convert the returned list into a list of #GtkTreeRowReferences.
+ * Creates a list of path of all selected rows.
+ *
+ * Additionally, if you are planning on modifying the model after calling
+ * this function, you may want to convert the returned list into a list
+ * of #GtkTreeRowReferences.
+ *
  * To do this, you can use gtk_tree_row_reference_new().
  *
  * To free the return value, use:
@@ -552,7 +555,7 @@ gtk_tree_selection_get_selected (GtkTreeSelection  *selection,
  * g_list_free_full (list, (GDestroyNotify) gtk_tree_path_free);
  * ]|
  *
- * Returns: (element-type GtkTreePath) (transfer full): A #GList containing a #GtkTreePath for each selected row.
+ * Returns: (element-type GtkTreePath) (transfer full): the selected paths
  *
  * Since: 2.2
  **/
